### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by davidedwardclark** [Abban/jQuery-Ajax-File-Upload#1]
Does this plugin work in IE9?

**Comment by Abban** [comment]
Yeah it should. I didn't do any testing myself but I integrated it with a project in work and it passed the IE tests.
